monitor/tasks.py

```
@@ -0,0 +1,489 @@
+        
+        response = client.chat.completions.create(
+            model="gpt-4o-mini",
+            messages=[{"role": "user", "content": prompt}],
+            max_tokens=800,
+            temperature=0.3
+        )
+        
+        return response.choices[0].message.content
+        
+    except Exception as e:
+        logger.error(f"OpenAI global summary generation failed: {str(e)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)